store/todos.js

```diff
@@ -67,11 +92,14 @@ export const mutations = {
       createdAt: new Date().toDateString(),
     };
 
-    state.taskList.push(task);
+    state.taskList = [task, ...state.taskList];
   },
   setCompleteRequest: (state, val) => {
     state.completeRequest = val;
   },
+  setTotalPage: (state, val) => {
```

Comment:
good to prefix unused values with `_` i.e. `_val`, otherwise this'll throw error in ts.

Reply:
noted

Reply:
Resolved